ui/src/__tests__/spec/tests/services.spec.js

```diff
@@ -82,35 +84,28 @@ describe('services screen tests', () => {
         await modalDeleteButton.click();
     });
 
-    describe('services screen tests', () => {
```

Comment:
duplicate 'describe' declaration removed, so most of the change here is just indentation fix